.github/workflows/blackduck_scan.yaml

```diff
@@ -1,5 +1,9 @@
-name: Blackduck Scan Cronjob
+name: Blackduck SCA Scan
 on:
+  #push:
+  #  branches: [ "main" ]
```

Comment:
are you sure?

Reply:
yes, this was added by me, but commented out. Once decision for blackduck has been done, we can uncomment it.